src/Globe/TileGeometry.js

```diff
@@ -232,7 +232,7 @@ TileGeometry.prototype.computeBuffers = function computeBuffers(params, builder)
             return id;
         };
 
-        buildUVSkirt = function buildUVSkirt() {
+        buildUVSkirt = function buildUVSkirt(id) {
```

Comment:
I'm curious: why this change? I'm not against it, but what was the linter saying?